app/src/main/java/com/celzero/bravedns/ui/SettingsFragment.kt

```diff
@@ -614,6 +647,36 @@ class SettingsFragment : Fragment(R.layout.activity_settings_screen) {
         alertDialog.show()
     }
 
+    private fun showDialogForNotificationAction() {
+        val alertBuilder = AlertDialog.Builder(requireContext())
+        alertBuilder.setTitle(getString(R.string.settings_notification_dialog_title))
+        val items = arrayOf(getString(R.string.settings_notification_dialog_option_1), getString(R.string.settings_notification_dialog_option_2), getString(R.string.settings_notification_dialog_option_3))
+        val checkedItem = persistentState.notificationAction
+        alertBuilder.setSingleChoiceItems(items, checkedItem) { dialog, which ->
+            dialog.dismiss()
+            if (persistentState.notificationAction != which) {
+                when (which) {
+                    0 -> {
+                        b.genSettingsNotificationDesc.text = getString(R.string.settings_notification_desc,getString(R.string.settings_notification_desc1) )
+                        persistentState.notificationAction = 0
```

Comment:
Please don't use raw ints. Always define them, either in `enums` or as plain old `constant`s.

Reply:
Removed usage of raw ints.